angular/authentication/README.md

```diff
@@ -61,7 +61,77 @@ function refreshTokenIfExpired () {
 ```
 
 
+# Authentication Token Interceptor
+
+Authentication token interceptor which will add token to request.
+
+
+```ts
+import {Injectable} from '@angular/core';
+import {HttpEvent, HttpHandler, HttpInterceptor, HttpRequest} from '@angular/common/http';
+import {Observable} from 'rxjs';
+
+import {environment} from '../../../environments/environment';
+
+@Injectable()
+export class JwtInterceptor implements HttpInterceptor {
+  constructor(private authenticationService: AuthenticationService) {
+  }
+
+  intercept(request: HttpRequest<any>, next: HttpHandler): Observable<HttpEvent<any>> {
+    // add auth header with jwt if user is logged in and request is to the api url
+    const currentUser = localStorage.currentUser;
+    const isApiUrl = request.url.startsWith(environment.baseUrl);
+
+    if (currentUser && isApiUrl) {
+      request = request.clone({
+        setHeaders: {
+          Authorization: `Bearer ${currentUser}`
+        }
+      });
+    }
+
+    return next.handle(request);
+  }
+}
+```
+
+# Authentication Error Interceptor
```

Comment:
Maybe this should be in Interceptor topic instead krub P'? What do you think krub?

Reply:
It is ok to move this to interceptor topic.